db/migrate/20220312153644_add_scoreboard_json_to_matches.rb

```diff
@@ -0,0 +1,5 @@
+class AddScoreboardJsonToMatches < ActiveRecord::Migration[7.0]
+  def change
+    add_column :matches, :scoreboard, :json
```

Comment:
```suggestion
    add_column :matches, :scoreboard, :jsonb
```
Para aprovechar postgres